src/assets/config_files/types_collabs_apps.json

```diff
@@ -395,6 +395,195 @@
       }
     ]
   },
+  "simulationandvalidationofacerebellarmulticompartmentalpurkinjecellmodel": {
```

Comment:
I think this item should be `simulationandvalidationofamousepurkinjecellmulticompartmentalmodel` because the usecase is called "Simulation and validation of a mouse Purkinje cell multicompartmental model".
Am I correct? (the current link relates to ...validation of a cerebellar multicompartmental ....)